CMake Error at ANTS.cmake:41 (find_package)

Hi All,
Im getting the following error during both Superbuild or manual build.
```-- Building ANTS version "v2.4.1.post7-g71c9890"
CMake Error at ANTS.cmake:41 (find_package):
  By not providing "FindITK.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "ITK", but
  CMake did not find one.

  Could not find a package configuration file provided by "ITK" with any of
  the following names:

    ITKConfig.cmake
    itk-config.cmake

  Add the installation prefix of "ITK" to CMAKE_PREFIX_PATH or set "ITK_DIR"
  to a directory containing one of the above files.```
I've attached the following files. Please advise.


[CMakeCache.txt](https://github.com/ANTsX/ANTs/files/9790046/CMakeCache.txt)
[cmake.log](https://github.com/ANTsX/ANTs/files/9790047/cmake.log)
[build.log](https://github.com/ANTsX/ANTs/files/9790048/build.log)
[CMakeOutput.log](https://github.com/ANTsX/ANTs/files/9790061/CMakeOutput.log)
[CMakeError.log](https://github.com/ANTsX/ANTs/files/9790062/CMakeError.log)

--- a/ANTS.cmake
+++ b/ANTS.cmake
@@ -163,6 +163,6 @@ endif()
 
 #Only install ITK/VTK libraries if shared build and superbuild is used
 if(BUILD_SHARED_LIBS AND ((NOT USE_SYSTEM_ITK) OR ((NOT USE_SYSTEM_VTK) AND USE_VTK)))
-  install(DIRECTORY ${CMAKE_BINARY_DIR}/../staging/lib/
+  install(DIRECTORY ${CMAKE_BINARY_DIR}/../staging/${CMAKE_INSTALL_LIBDIR}/
           DESTINATION ${CMAKE_INSTALL_LIBDIR})
 endif()

--- a/SuperBuild/External_ITKv5.cmake
+++ b/SuperBuild/External_ITKv5.cmake
@@ -178,7 +178,7 @@ if(NOT DEFINED ${extProjName}_DIR AND NOT ${USE_SYSTEM_${extProjName}})
     DEPENDS
       ${${proj}_DEPENDENCIES}
   )
-  set(${extProjName}_DIR ${CMAKE_BINARY_DIR}/staging/lib/cmake/${ITK_VERSION_ID})
+  set(${extProjName}_DIR ${CMAKE_BINARY_DIR}/staging/${CMAKE_INSTALL_LIBDIR}/cmake/${ITK_VERSION_ID})
 else()
   if(${USE_SYSTEM_${extProjName}})
     find_package(${extProjName} ${ITK_VERSION_MAJOR} REQUIRED)

